src/FilenameParsing/AbstractFileIDHelper.php

```diff
@@ -2,12 +2,28 @@
 
 namespace SilverStripe\Assets\FilenameParsing;
 
+use SilverStripe\Core\Convert;
 use SilverStripe\Core\Injector\Injectable;
 
 abstract class AbstractFileIDHelper implements FileIDHelper
 {
     use Injectable;
 
+    /**
+     * A variant type for encoding a variant filename with a different extension than the original.
+     */
+    public const EXTENSION_REWRITE_VARIANT = 'extRewrite';
+
+    /**
+     * use the original file's extension
+     */
+    private const EXT_ORIGINAL = 0;
+
+    /**
+     * use the variant file's extension
+     */
+    private const EXT_VARIANT = 1;
```

Comment:
```suggestion
    /**
     * Use the original file's extension
     */
    private const EXTENSION_ORIGINAL = 0;

    /**
     * Use the variant file's extension
     */
    private const EXTENSION_VARIANT = 1;
```
Make the naming consistent with other contant above and also make it slightly clearer. Also initial caps for "Use"

Will also need to update references to consts

Reply:
Done